certification/internal/bundle/bundle.go

```diff
@@ -18,17 +18,27 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const ocpVerV1beta1Unsupported = "4.9"
-
 // versionsKey is the OpenShift versions in annotations.yaml that lists the versions allowed for an operator
 const versionsKey = "com.redhat.openshift.versions"
 
+// This table signifies what the NEXT release of OpenShift will
+// deprecate, not what it matches up to.
+var ocpVerV1beta1Unsupported = map[string]string{
```

Comment:
Should we rename this variable? It doesn't mean anything to me right now. Maybe I'm missing something

Reply:
Valid.